packages/lesmis/src/views/DashboardReportTabView.js

```diff
@@ -146,32 +146,36 @@ export const DashboardReportTabView = ({ entityCode, TabSelector }) => {
       <DashboardSection ref={topRef}>
         <FetchLoader isLoading={isLoading} isError={isError} error={error}>
           {data &&
-            Object.entries(data).map(([heading, dashboardGroup]) => (
-              <TabPanel key={heading} isSelected={heading === selectedDashboard}>
-                {Object.entries(dashboardGroup).map(([groupName, groupValue]) => {
+            data.map(dashboard => (
+              <TabPanel
+                key={dashboard.dashboardName}
+                isSelected={dashboard.dashboardName === selectedDashboard}
+              >
+                {(() => {
```

Comment:
Inline js is expected to be a single resolvable statement, so since it's doing two steps (a filter followed by a ternary return) it needs to be wrapped. Previously it was wrapped in the `map` statement, but the changes in the layout of the `data` object mean that `map` is no longer necessary.